src/test/java/core/basesyntax/service/transaction/BalanceTransactionHandlerTest.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.service.transaction;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+public class BalanceTransactionHandlerTest {
+    private static final TransactionHandler transactionHandler = new BalanceTransactionHandler();
```

Comment:
initialize this service in a method annotated with Before or BeforeClass (depends on the version of junit)